README.md

```diff
@@ -1 +1,17 @@
-# MoolahProject
\ No newline at end of file
+# MoolahProject
+
+## Installation
+- Install Unreal Engine 4.27 from the Epic Games Launcher
+- Download this Project
+- Open the PAYDAY3.UProject file in the Folder you downloaded it
+
+
+##  Requirements / Prerequisites
+ ###  Prerequisites
+       - EGS Launcher
+       - Unreal Engine 4.27
+       - Around 30GB of Storage Space for the Engine and Base mod files
```

Comment:
Remove this indentation, so this doesn't render as a code block.

Reply:
Yea that should I didnt know how to remove it since I havent used Github in a long time, so I didnt remove it